### DISCUSSION TITLE
Allow "ReadOnly" mode for GitHub

### DISCUSSION THREAD

**Discussion by stramel** [six7/figma-tokens#1311] [Ideas]
Some users would only be allowed to read from a GitHub repo. It would be nice to hide the "Update" button when the authorized user doesn't have `push: true`.

https://docs.github.com/en/rest/repos/repos#get-a-repository

```
"permissions": {
    "pull": true,
    "push": false,
    "admin": false
  },
  ```

**Comment by six7** [comment]
This should already be the case, is it not for you?

**Reply by stramel** [reply]
Ah, you're correct. Sorry for the noise.
